pkg/collector/py/datadog_agent.go

```diff
@@ -173,7 +173,7 @@ func GetSubprocessOutput(argv **C.char, argc, raise int) *C.PyObject {
 
 	length := int(argc)
 	subprocessArgs := make([]string, length-1)
-	cmdSlice := (*[1 << 30]*C.char)(unsafe.Pointer(argv))[:length:length]
+	cmdSlice := (*[1<<29 - 1]*C.char)(unsafe.Pointer(argv))[:length:length]
```

Comment:
just a note on the previous value: seems like `1<<30` was used here because that's what the link in the comment above (https://github.com/golang/go/wiki/cgo#turning-c-arrays-into-go-slices) uses in its example. Should be safe to change it to `1<<29-1` (see https://github.com/docker/docker-credential-helpers/pull/61 for a nice explanation on why this value makes sense)